lib/ui/compositing/scene_builder.cc

```diff
@@ -189,8 +189,7 @@ void SceneBuilder::pushShaderMask(Dart_Handle layer_handle,
                                  maskRectBottom);
   auto sampling = ImageFilter::SamplingFromIndex(filterQualityIndex);
   auto layer = std::make_shared<flutter::ShaderMaskLayer>(
-      shader->shader(sampling)->skia_object(), rect,
-      static_cast<SkBlendMode>(blendMode));
+      shader->shader(sampling), rect, static_cast<SkBlendMode>(blendMode));
```

Comment:
We should have SML take a DlBlendMode now that we're converting it to DlColorSource. I believe it is the same cast, just with `s/Sk/Dl/` (I think there is a test case that confirms 1:1 of Sk/Dl blend modes)

Reply:
Done